src/debug.c

```diff
@@ -2305,6 +2301,12 @@ void applyWatchdogPeriod(void) {
     }
 }
 
+void debugPauseProcess(void) {
+    serverLog(LL_DEBUG, "Process is about to stop.");
+    raise(SIGSTOP);
+    serverLog(LL_DEBUG, "Process has been continued.");
```

Comment:
```suggestion
    serverLog(LL_NOTICE, "Process is about to stop.");
    raise(SIGSTOP);
    serverLog(LL_NOTICE, "Process has been continued.");
```

loglevel is set to notice except for TLS tests.